internal/enginecache/cache.go

```diff
@@ -8,24 +8,28 @@ import (
 	"io/ioutil"
 	"net/http"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/otiai10/copy"
+	. "github.com/pkg/errors"
+
+	"github.com/go-flutter-desktop/hover/internal/build"
 	"github.com/go-flutter-desktop/hover/internal/log"
-	"github.com/pkg/errors"
 )
 
 func createSymLink(oldname, newname string) error {
 	err := os.Remove(newname)
 	if err != nil && !os.IsNotExist(err) {
-		return errors.Wrap(err, "failed to remove existing symlink")
+		return Wrap(err, "failed to remove existing symlink")
```

Comment:
Nit: leave the package name (`errors`) in front of the methods.

Reply:
That was some refactoring done by the IDE. It wasn't intentionally